Server/src/services/DataSetService.ts

```diff
@@ -129,6 +133,13 @@ export class DataSetService {
       rawData = await this.dataQuery.getDatasetIDFromCategory(categoryReceived);
       rawDatasetIds = rawDatasetIds.concat(await this.createDatasetIdArray(rawData));
     }
+    if (datapointsReceived) {
+      for (let i = 0; i < datapointsReceived.length; i++) {
+        paramsEntered++

```

Comment:
Is there a reason we are tracking paramsEntered?

Reply:
paramsEntered is needed by the selectDatasetIds method that figures out what the "common" data set IDs are amongst all the various search terms.